_guide/_pages/interviews/technical.md

```diff
@@ -89,7 +89,7 @@ If the candidate is applying for a specific technical role, ask the relevant pos
     </button>
   </h4>
     <div id="consulting" class="usa-accordion__content">
-        <a href="https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.8lizmty3jisl">Please explain a complex technical concept or tool to me as if I were a non-technical partner</a>
+        <a href="https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.8lizmty3jisl">Pick a foundational web development concept or technology and explain it at two levels: first as you would to a colleague who’s not a software developer, like a designer or product manager; next, as you would to a peer.</a>
```

Comment:
**Praise:** Oooh I ❤️ this so much, definitely leveling up this question!

Reply:
All credit to our old colleague Jacob Kaplan-Moss,[ who wrote about this earlier this year!](https://jacobian.org/2021/feb/8/interview-questions-explain-a-topic/)